server/MyDonor/MyDonor.Domain/Models/Feedback.cs

```diff
@@ -1,6 +1,7 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
```

Comment:
Move the `using System.ComponentModel.DataAnnotations.Schema;` import to global usings.